src/oauth2.js

```diff
@@ -1,38 +1,38 @@
 'use strict';
-const {urlParams} = require('./utils');
+const { urlParams } = require('./utils');
 const request = require('./request');
 
-module.exports = function(config)
-{
+module.exports = function (config) {
 	const auth_url = config.auth_url;
 	const token_url = config.token_url;
-	const client_id = config.client_id;
+	var client_id = config.client_id;
```

Comment:
If this is not reassigned why is it now a var? @MuhammadAliGhaffarr

Reply:
@MuhammadAliGhaffarr I can understand that client_id will override. But I don't see the code change where it gets reassigned.